guides/common/modules/proc_viewing-vulnerabilities.adoc

```diff
@@ -0,0 +1,20 @@
+[id="viewing-vulnerabilities"]
+= Viewing vulnerabilities
+
+You can view Common Vulnerabilities and Exposures (CVE) from the {advisorengine}.
+You can examine how the CVEs affect your hosts.
```

Comment:
Do we want to give more details here: that we have the CVEs list page and that we have the CVE list tab for a specific host?

Reply:
@jeremylenz Update how? Will we have a second attribute like {vulnerabilityengine} for the Vulnerability IoP service? Or will we change the value of {advisorengine} to something that will cover both services?

Reply:
I think it will be an additional attribute for the Vulnerability service.